.travis.yml

```diff
@@ -6,6 +6,9 @@ addons:
 jdk:
 - oraclejdk8
 install: true
+before_script:
```

Comment:
The integration test that requires this capability is for the new `SwingWorkerCompletionWaiter` class, which _is_ pure Swing code.  However, to avoid introducing this requirement, I will change the integration test to a unit test that uses a test double for `SwingWorker`.